### PR TITLE
Move v2v job

### PR DESCRIPTION
**Pull request by pkliczewski**
As part of unification of import disk and v2v abps we need to have
the scripts located here so we can share template definitions.

### CODE REVIEW

**Comment by pkliczewski** [comment]
@aglitke please take a look

File: bin/run.sh [hunk 1]
```diff
@@ -0,0 +1,216 @@
+#!/bin/bash -x
```
Comment (aglitke) [inline bin/run.sh]: Please let's name this run-v2v.sh so we can distinguish between v2v and url scripts

File: images/v2v-job/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM fedora:26
```
Comment (aglitke) [inline images/v2v-job/Dockerfile:1]: Does it make sense to have a Makefile target (at the top level) for building this image?  The reason I ask is when we add the templates/ dir at the top level also we will need to exec this build there using 
```
  docker build -f images/v2v-job/Dockerfile .
```

**Review by aglitke — COMMENTED** [review]
Looks exactly like what I had in mind.  Minor nits and let's merge this soon.

File: bin/run.sh (hunk 1 above)
Comment (pkliczewski) [reply bin/run.sh]: OK

File: images/v2v-job/Dockerfile (hunk 2 above)
Comment (pkliczewski) [reply images/v2v-job/Dockerfile:1]: Agreed, will move Makefile as well

File: bin/job [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  export VMNAME=$(echo $SRC | sed -r 's/[_.]+/-/g')
+
+  # get resources from vm xml
+  export CORES=$(xmllint --xpath "/domain/vcpu/text()" $DOMXML)
+  export MEMORY=$(xmllint --xpath "/domain/memory/text()" $DOMXML)
+  export UNIT=$(xmllint --xpath "string(/domain/memory/@unit)" $DOMXML)
+  export BUS=$(xmllint --xpath "string(/domain/devices/disk/target/@bus)" $DOMXML)
+
+  # Create job offlinevirtualmachine
+  j2 template/default-vm.j2 | oc create -f
```
Comment (pkliczewski) [inline bin/job]: @aglitke How do you want to customize which template to use?

File: Makefile [hunk 4]
```diff
@@ -1,6 +1,7 @@
 DOCKERHOST = docker.io
 DOCKERORG = import-vm-apb
 IMAGENAME = import-vm
+QUAY_ORG = kubevirt
```
Comment (pkliczewski) [inline Makefile:4]: @fabiand Will you be able to configure build process in Quay so anytime we merge (after this PR) we will rebuild the image?

**Review by aglitke — APPROVED** [review]

**Pull request merged**